schema.prisma

```diff
@@ -1004,3 +1005,64 @@ enum EmailType {
 
   @@map("email_type")
 }
+
+/// CampaignApplication represents a request for a new campaign - it is not a Campaign yet and has to proove it needs to be
+model CampaignApplication {
+  id        String    @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  createdAt DateTime  @default(now()) @map("created_at") @db.Timestamptz(6)
+  updatedAt DateTime? @updatedAt @map("updated_at") @db.Timestamptz(6)
+
+  // organizer editable fields
+  // need to be logged in to create a campaign application
+  organizerId             String                    @map("organizer_id") @db.Uuid
+  organizer               Organizer?                @relation(fields: [organizerId], references: [id])
+  organizerName           String                    @map("organizer_name") @db.VarChar(200)
+  organizerEmail          String?                   @unique @map("organizer_email") @db.Citext
+  organizerPhone          String?                   @map("organizer_phone") @db.VarChar(50)
+  // will create a beneficiary after approve of campaign application
+  beneficiary             String                    @db.VarChar(1500)
+  organizerBeneficiaryRel String                    @map("organizer_beneficiary_relationship") @db.Text
+  campaignName            String                    @map("campaign_name") @db.VarChar(200)
+  goal                    String                    @db.Text
+  history                 String?                   @db.Text
+  amount                  String                    @db.VarChar(200)
```

Comment:
Yes - all these fields are closely matched to the current google form. 

My understanding is that the amount here is more of a discussion point that actual money that will have transactions and calculations derived from it.

It might be 500lv or 15456EUR or whatever